modules/plugin/epsg-hsql/src/test/java/org/geotools/referencing/factory/epsg/hsql/PropertyDumper.java

```diff
@@ -40,48 +48,123 @@ public static void main(String[] args) throws Exception {
 
         try (FileOutputStream out = new FileOutputStream(filename);
                 Writer writer = new BufferedWriter(new OutputStreamWriter(out, "8859_1"))) {
-            writer.append("Generate from EPSG database version " + ThreadedHsqlEpsgFactory.VERSION);
 
-            Properties props = new Properties();
-            List<String> codes = new ArrayList<>(CRS.getSupportedCodes("EPSG"));
-            Collections.sort(codes, (c1, c2) -> {
+            if (LEGACY) {
+                legacyDump(out, writer, diff);
+            } else {
+                sortedDump(writer, diff);
+            }
```

Comment:
Do we need both versions and why?

Reply:
Actually nope. we can get rid of the legacy one. I was using it to produce the list as if it was before the sorting that I have added (which is helpful for quick comparisons).
I'm removing it (we can eventually get the previous version from git history)